app/controllers/api/subcollections/alert_status_states.rb

```
@@ -0,0 +1,49 @@
+module Api
+  module Subcollections
+    module AlertStatusStates
+      def alert_status_states_query_resource(object)
+        object.miq_alert_status_states
+      end
+
+      def alert_status_states_add_resource(object, _type, _id, data = nil)
+        alert_status_state = create_alert_status_state(data)
```

It looks too like this could silently fail if `#create` is not successful